bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/DialogProcessor.java

```diff
@@ -376,4 +384,17 @@ protected void say(@Nullable String text) {
             }
         }
     }
+
+    /**
+     * Check if other DialogProcessor instance have same configuration ignoring the configured keyword
+     *
+     * @param dialogProcessor Other DialogProcessor instance
+     */
+    public boolean isCompatible(DialogProcessor dialogProcessor) {
+        return this.sink.equals(dialogProcessor.sink) && this.source.equals(dialogProcessor.source)
+                && Objects.equals(this.ks, dialogProcessor.ks) && this.stt.equals(dialogProcessor.stt)
```

Comment:
What is the difference leading to use `Objects.equals` for ks ?

Reply:
Because both are nullable, but it was an error as I pretended to ignore all the keyword spotting related config in this methods.